docs/setup/fault_tolerance.md

```diff
@@ -45,9 +45,6 @@ To enable checkpointing, call `enableCheckpointing(n)` on the `StreamExecutionEn
 
 Other parameters for checkpointing include:
 
-- *Number of retries*: The `setNumberOfExecutionRerties()` method defines how many times the job is restarted after a failure.
```

Comment:
Removing from here because it will be taken care in https://github.com/apache/flink/pull/2951/files